ui/src/main/resources/Meeting/Code/WebHomeSheet.xml

```diff
@@ -51,6 +51,7 @@ $services.localization.render('contrib.meeting.home.presentation', ['[[**', '**&
   'leader': {"type":"text","size":10,"html":true},
   'place': {"type":"text","size":10},
   'startDate': {"type":"text","size":10},
+  'endDate': {"type":"hidden"},
```

Comment:
As an improvement, separate from this PR, you can update the livetable definition to use the recommended code style: replacing double quotes `"` with single quotes `'` and adding a blank space after colon `:`. So you will have for this line
`'endDate': {'type': 'hidden'},`